stdpopsim/genomes.py

```diff
@@ -65,4 +66,8 @@ def recombination_map(self, map_name=None):
         if map_name is None:
             map_name = self.default_genetic_map
         genetic_map = genetic_maps.get_genetic_map(self.species, map_name)
-        return genetic_map.get_chromosome_map(self.name)
+        try:
+            ret = genetic_map.get_chromosome_map(self.name)
+        except FileNotFoundError:
+            ret = msprime.RecombinationMap.uniform_map(self.length, 0)
```

Comment:
I think this should produce a warning, because there's any number of things that could go wrong, so we don't want to silently substitute a zero-recomb map. Like: "Warning: recombination map not found for chromosome {}, substituting a zero-recombination map."

Reply:
There's two things here:
1. If it's a string that doesn't correspond to any know chromosome (name="blahblah") then it should raise a ValueError
2. If it is a known chromosome and we don't have a map, then we insert a zero map.

Should we still raise a warning in case 2? I guess it's safer, in case someone did something like deleting all the recombination map files but didn't remove the map directory.

Reply:
OK, to sum up, it seems I should:

1. add a method, say, `contains_chromosome_map(self, name)` to `GeneticMap` that returns a boolean flag indicative of whether or not it contains a recombination_map for a certain chromosome.

2. In `Chromosome.recombination_map(self, mape_name=None)` we first check that a map exists,
if so, return the recombination_map
if not, raise a warning & return the flat map

3. in `GeneticMap.get_chromsome_map(self, name)` 
We check if the name exists, and throw ValueError if not.

Reply:
I think we should raise a warning if we default to zero recombination.

It seems like for step (1) we need a way of recording/standardizing what the legal chromosome names are (grrr, "chr1" versus "chr01" versus "1").

Reply:
We've touched on the idea of synonyms in another issue @petrelharp (https://github.com/popgensims/stdpopsim/issues/16), so this can be handled as part of that.